### PR TITLE
Add app.json

### PR DESCRIPTION
**Pull request by stevenwilkin**
I've added Logentries to staging and production though I don't have the `ggp-accounts` login to select the $9 tier for production.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "addons": [
+    "logentries:entry_plus"
```
Comment (andrewgarner) [inline app.json]: I would just specify "logentries" and let Heroku choose the free tier initially. That has the effect of allowing you to spin up a new environment with suitable dependencies - you can then choose to scale those as you see fit.

File: app.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  ],
+  "buildpacks": [
+    {
+      "url": "https://github.com/kr/heroku-buildpack-go.git"
```
Comment (andrewgarner) [inline app.json]: Heroku [now officially support](http://blog.heroku.com/archives/2015/7/7/go_support_now_official_on_heroku) Go and have [their own buildpack](https://github.com/heroku/heroku-buildpack-go). I think we can remove this and let Heroku automatically detect the language.

Comment (stevenwilkin) [inline app.json]: We're using kr's buildpack:

```
$ heroku config --remote production | grep BUILDPACK
BUILDPACK_URL:         https://github.com/kr/heroku-buildpack-go.git
```

File: app.json (hunk 1 above)
Comment (stevenwilkin) [inline app.json]: Good idea, I didn't know Heroku would choose the free tier if we aren't explicit about it

File: app.json (hunk 2 above)
Comment (andrewgarner) [inline app.json]: We should use the official buildpack now.

Comment (stevenwilkin) [inline app.json]: Makes sense, that's great that Go is now officially supported

**Pull request merged**